Lib/ntpath.py

```diff
@@ -17,6 +17,7 @@
 defpath = '.;C:\\bin'
 devnull = 'nul'
 
+import _winapi
```

Comment:
Bytes paths need to be supported, and a fallback is needed when ntpath is used in POSIX. One issue is whether we want to continue to use a simple ASCII lower case for bytes, or decode as a string first via `os.fsdecode()` to use a proper lower case. I think it should implemented properly.

```suggestion
try:
    import _winapi
    
    def _normcase(s):
        is_bytes = isinstance(s, bytes)
        if is_bytes:
            s = os.fsdecode(s)
        s = _winapi.LCMapStringEx(_winapi.LOCALE_NAME_INVARIANT,
                                  _winapi.LCMAP_LOWERCASE,
                                  s.replace('/', '\\'))
        if is_bytes:
            s = os.fsencode(s)
        return s

except ImportError:
    def _normcase(s):
        if isinstance(s, bytes):
            return s.replace(b'/', b'\\').lower()
        return s.replace('/', '\\').lower()

def normcase(s):
    """Normalize case of pathname.
    
       Makes all characters lowercase and all slashes into backslashes.
    """
    return _normcase(os.fspath(s))
```

Reply:
Ah, never thought this module can be used in POSIX systems...